bindings/nodejs/lib/utils/utils.ts

```diff
@@ -142,14 +142,14 @@ export class Utils {
     static computeTokenId(
         aliasId: AliasId,
         serialNumber: number,
-        tokenSchemeType: TokenSchemeType,
+        tokenSchemeKind: TokenSchemeType,
```

Comment:
Why name it now `kind`?

Reply:
Because that's the name of the field in the `UtilsMethod`

Reply:
At least it was when I made the change...

Reply:
ComputeFoundryId still uses `kind`

Reply:
You're doing the change in this PR
Shouldn't we rather change ComputeFoundryId?
In Rust we only use `kind` because `type` is not allowed as field name

Reply:
What else uses `kind`? I only see it in `ComputeFoundryId`
I would use `type` everywhere bindings related

Reply:
the python bindings

Reply:
Changed them all